### PR TITLE
Clarify 4-element binning args

### PR DESCRIPTION
**Pull request by oerc0122**
Improve clarity of meaning for 4th binning arg as reported as confusing by Christian

### CODE REVIEW

**Review by abuts — COMMENTED** [review]
should be reconciled with Re #1508 which havily modified text of this document. Should be merged ASAP then to avoid merge conflicts in nearest future.

File: documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst [hunk 1]
```diff
@@ -264,7 +264,7 @@ Alternatively you may define some parameters in constructor, and then set other
    proj.offset = [1,0,0];
 
 
-The most important fields of ``line_proj`` constructor are the fields, which define the 
+The most important fields of ``line_proj`` constructor are the fields, which define the
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst:267]:
```suggestion
The most important fields of ``line_proj`` constructor are the fields which define the
```
with this comma, there is no intent to limit the scope of fields to the defining scope. The which clause is merely an explanation of the term fields. The comma must be removed to limit the scope

File: documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst [hunk 2]
```diff
@@ -422,22 +422,22 @@ projection in a similar way to other projections, but instead use ``sphere_proj`
           label: {'|Q|'  '\theta'  '\phi'  'En'}
           title: ''
 
-Projection's lattice parameters (``alatt`` and ``angdeg``) are taken within the ``cut`` from source ``sqw`` object. 
+Projection's lattice parameters (``alatt`` and ``angdeg``) are taken within the ``cut`` from source ``sqw`` object.
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst:425]:
```suggestion
The projection's lattice parameters (``alatt`` and ``angdeg``) are taken within the ``cut`` from source ``sqw`` object.
```

File: documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst [hunk 3]
```diff
@@ -422,22 +422,22 @@ projection in a similar way to other projections, but instead use ``sphere_proj`
...
+The projection defines spherical coordinates system, where :math:`\theta` angle is
+measured from z-axis directed along :math:`e_z` vector of the
 projection and changes from :math:`0` to :math:`180^o`. :math:`\phi` angle is measured
-from :math:`r_x` vector of the projection and changes from :math:`-180^o` to :math:`180^o`. 
+from :math:`r_x` vector of the projection and changes from :math:`-180^o` to :math:`180^o`.
 Spherical projection :math:`r_x` vector corresponds to ``sphere_proj`` class :math:`e_x` vector
 for orthogonal reciprocal lattice. If reciprocal lattice is non-orthogonal, the :math:`e_x`-vector
-of spherical projection defines the :math:`z-x` plane of spherical coordinate system and :math:`r_x` 
+of spherical projection defines the :math:`z-x` plane of spherical coordinate system and :math:`r_x`
 vector is build in this plane but orthogonal to :math:`e_z` vector.
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst:435]:
```suggestion
vector is built in this plane but orthogonal to :math:`e_z` vector.
```

File: documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst [hunk 4]
```diff
@@ -572,11 +572,11 @@ projection axes and binning:
 
 Legacy calls to ``cut``: ``cut_sqw`` and ``cut_dnd``
 ----------------------------------------------------
-Historically, ``cut`` was used in two different forms ``cut_sqw`` and ``cut_dnd``. These forms 
+Historically, ``cut`` was used in two different forms ``cut_sqw`` and ``cut_dnd``. These forms
 are still available now. ``cut_sqw`` is fully equivalent to ``cut`` except attempt to apply it to
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst:576]:
```suggestion
are still available now. ``cut_sqw`` is fully equivalent to ``cut``, except an attempt to apply it to
```

File: documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst [hunk 5]
```diff
@@ -572,11 +572,11 @@ projection axes and binning:
...
-``dnd`` object or file, containing ``dnd`` object will throw ``HORACE:DnDBase:invalid_argument`` exception. 
-``cut_dnd`` is equivalent to ``cut`` applied to ``dnd`` object or correspondent file. If ``cut_dnd`` is applied 
-to ``sqw`` object, it acts on ``dnd`` part of ``sqw`` object (``sqw.data`` field) and returns ``dnd`` 
+``dnd`` object or file, containing ``dnd`` object will throw ``HORACE:DnDBase:invalid_argument`` exception.
+``cut_dnd`` is equivalent to ``cut`` applied to ``dnd`` object or correspondent file. If ``cut_dnd`` is applied
```
Comment (cmarooney-stfc) [inline documentation/user_docs/docs/manual/Cutting_data_of_interest_from_SQW_files_and_objects.rst:578]:
```suggestion
``cut_dnd`` is equivalent to ``cut`` applied to ``dnd`` object or corresponding file. If ``cut_dnd`` is applied
```

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

**Pull request merged**